charts/hlf-k8s/templates/deployment-toolbox.yaml

```diff
@@ -116,15 +116,9 @@ spec:
       - name: tls
         secret:
           secretName: {{ $.Values.secrets.tls }}
-      - name: tls-rootcert
-        secret:
-          secretName: {{ $.Values.secrets.tlsRootCert }}
       - name: tls-client
         secret:
           secretName: {{ $.Values.secrets.tlsClient }}
-      - name: tls-clientrootcert
-        secret:
-          secretName: {{ $.Values.secrets.tlsClientRootCerts }}
```

Comment:
I'm not sure we should modify it here.
In our specific case it will work, but there can be some case where tls-rootcert != tls-clientrootcert

Reply:
Under which scenario would that be the case ?

Reply:
The choice can be done by the user.
In our case we use the same, but we could ask to the ca to generate un cacert for the tls

Reply:
https://openblockchain.readthedocs.io/en/latest/API/MemberServicesAPI/#tls-certificate-authority

Reply:
`tls-rootcert` and `tls-clientrootcert` need to be bring it back